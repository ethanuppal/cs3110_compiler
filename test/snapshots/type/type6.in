func foo() {
    let x = 1
}
func main() {
    let x = 1
    print x
}
