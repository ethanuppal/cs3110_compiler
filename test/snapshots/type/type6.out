func foo() -> Unit {
    let x: Int = 1
    return
}
func main() -> Unit {
    let x: Int = 1
    print x
    return
}
